browser/css/jsdialogs.css

```diff
@@ -541,6 +541,11 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 	white-space: nowrap;
 }
 
+#ServerAuditDialog .ui-treeview-entry {
+	display: grid;
+	grid-template-columns: auto 1fr 180px;
+}
```

Comment:
I see 2 problems here:

* I would prefer to avoid hard coded pixels and instead use ratios
* Where I'm missing a pixel value is on the initial column (the icon)

```css
grid-template-columns: var(--btn-img-size-s) 6fr 3fr;
```

Reply:
Changed from pixel to ratios and used --btn-img-size-m as the ratio for the first column